masonry/src/widget/label.rs

```diff
@@ -55,14 +55,15 @@ impl Label {
             line_break_mode: LineBreaking::Overflow,
             show_disabled: true,
             brush: crate::theme::TEXT_COLOR.into(),
-            skip_pointer: false,
+            interactive: true,
         }
     }
 
-    // TODO - Rename
-    // TODO - Document
-    pub fn with_skip_pointer(mut self, skip_pointer: bool) -> Self {
-        self.skip_pointer = skip_pointer;
+    /// Sets the value returned by [`accepts_pointer_interaction`].
```

Comment:
Hmm, I'm surprised that this doc resolves? Is there a rustdoc feature I didn't know about (automatic handling of trait methods?)

It isn't shown as a valid link by rust-analyzer. I've not tried it in rustdoc

Reply:
Nope, rustdoc doesn't resolve it either.